airflow/utils/dag_processing.py

```diff
@@ -655,6 +693,7 @@ def start(self):
             # Update number of loop iteration.
             self._num_run += 1
 
+            simple_dags = self.collect_results()
```

Comment:
Was there a reason you moved the position of this call?

Reply:
Yes. First we need to create processes, and then we can read the results. Otherwise, we will never get the value on the first iteration of the loop. Other solution, we can increase loop iteration nunber: https://github.com/apache/airflow/blob/cb455dc81162680f90edcd78400e1ef46c09766d/tests/utils/test_dag_processing.py#L291

Reply:
Cool, sounds good as you have it